pkg/platform/kube/functionres/lazy.go

```diff
@@ -59,6 +59,7 @@ const (
 	containerMetricPortName       = "metrics"
 	nvidiaGpuResourceName         = "nvidia.com/gpu"
 	nginxIngressUpdateGracePeriod = 5 * time.Second
+	DefaultTargetCPU              = 75
```

Comment:
Why is this not in abstract platform as well?
It can be applied to local platform (if we want to support it)
Suggest to keep the deployment defaults in one place